marketplacetests/marketplace/app.py

```diff
@@ -66,7 +66,9 @@ def offline_message_text(self):
 
     def wait_for_notification_message(self, message):
         element = self.marionette.find_element(*self._notification_locator)
-        Wait(self.marionette).until(lambda m: element.text == message)
+        Wait(self.marionette).until(lambda m: element.text == message,
+                                    message='Notification not found. Expected: %s | Found: %s'
+                                            % (message, element.text))
```

Comment:
Oh, I see! I was thinking that the message would be constructed when it is needed, i.e., when the `Wait` times out, but I can see that that is not how it would work, and it would construct the message when first calling the `until`. Thanks @davehunt!